README.md

```diff
@@ -1,189 +1,64 @@
-| [Mobiconf](https://2017.mobiconf.org/) | **October 5-6, 2017** | 🇵🇱 Krakow, Poland |
-| [Release Notes](https://2017.releasenotes.tv/) | **October 16-18, 2017** | 🇺🇸 Chicago, IL, USA |
-| [Mobilization](http://2017.mobilization.pl/) | **October 21, 2017** | 🇵🇱 Łódź, Poland |
-| [Mobius](https://mobiusconf.com/en/) | **November 11, 2017** | 🇷🇺 Moscow, Russia |
-| [DevFest Baltics](https://devfest.gdg.lv/) | **November 17, 2017** | 🇱🇻 Riga, Latvia |
-| [UA Mobile](http://www.uamobile.org/) | **November 25, 2017** | 🇺🇦 Kyiv, Ukraine |
-| [MobOS](http://romobos.com/) | **February 15-16, 2018** | 🇷🇴 Cluj-Napoca, Romania |
-| [MobileTech](https://mobiletechcon.de/en/) | **March 12-14, 2018** | 🇩🇪 Munich, Germany | n/a |
-| [Appdevcon](http://appdevcon.nl/) previously called mDevcon | **March 15-18, 2018** | 🇳🇱 Amsterdam, Netherlands |
-| [Code Mobile](http://www.codemobile.co.uk) | **April 2-5, 2018** | 🇬🇧 Chester, UK |
-| [App Builders](https://www.appbuilders.ch/) | **April 16-17, 2018** | 🇨🇭 Lugano, Switzerland |
-| [Craft Conference](http://craft-conf.com) | **May 8-11, 2018** | 🇭🇺 Budapest, Hungary |
-| [MCE](http://2018.mceconf.com) | **June 5-6, 2018** | 🇵🇱 Warsaw, Poland | 
-| [ADDC - App Design & Development Conference](https://addconf.com/) | **July 4-6, 2018** | 🇪🇸 Barcelona, Spain |
-| [Mobiconf](https://2018.mobiconf.org/) | **October 4-5, 2018** | 🇵🇱 Krakow, Poland |
-| [Mobile Era](https://mobileera.rocks/) | **November 1-2, 2018** | 🇳🇴 Oslo, Norway |
-| [DevFest Baltics](https://devfest.gdg.lv/) | **November 16** | 🇱🇻 Jurmala, Latvia |
-| [Functional Swift Conference 2018](http://2018.funswiftconf.com) | **December 1, 2018** | 🇺🇸 New York, NY |
-
-### More
-
-* A [Twitter list](https://twitter.com/NeoNacho/lists/cocoaconferences) with all the conferences made by [Boris Bügling](https://twitter.com/NeoNacho)
-
-If you want to add a conference to this list or edit the info, send a **pull request**, [ping me on twitter](https://twitter.com/lascorbe) or get in touch at [luis@nsspain.com](mailto:luis@nsspain.com). Thank you!
+[http://ascor.be/CocoaConferences/](http://ascor.be/CocoaConferences/)
+=========================================================
+
+List of Cocoa conferences for iOS & macOS developers.
+
+## Adding a new conference
+
+The site is generated from [this file](https://github.com/lascorbe/CocoaConferences/_data/conferences.yml); add new conferences there in YAML format.
+
+When editing an existing conference, there is no need to manually remove the corresponding date fields for past conferences; those are updated automatically. Your entry should look like this:
+
+```yaml
+- name: The name of the conference
+  link: The URL for the conference website
+  location: Text briefly describing where the conference will be held
+  start: The day the conference starts (in "year-month-day" format)
+  end: The day the conference ends (in "year-month-day" format)
+  cocoa: true or false
```

Comment:
I believe it'll be more clear if this parameter is called `cocoa-only` rather than just `cocoa`

```suggestion
  cocoa-only: Specifies if the conference is just about Apple technologies (Swift, Objc, iOS, macOS, etc), or more generic but still relevant for Cocoa Developers (i.e. 2 platforms iOS & Android, mobile dev & design, etc). Accepted values are `true` or `false`
```

Reply:
👍